utopian-io/v2.utopian.io
# Adding tags to an article

### Verify that
Git checklist

Front end checklist

API checklist

Tests and comments checklist

---


### **Test plan**

Add tags to an article

* Go to the contribution page
* Start typing the tag you want for your article in the Tags input
* If any other article has a tag that shares the same 2 initial characters, it will give you the option for autocomplete, and how many articles have it.
* Tags starting with "first" and "dev" is already in the dev DB. If you type "fi" and 'de', you would be able to see those as options for the autocomplete.
* You can add a new one, or select an option from the list.
* Click on 'Update'.

**Expected result:** You should receive a message that your article was successfully published. You also can edit the tags after the article was published.
